client/__tests__/e2e/e2e.test.js

```diff
@@ -154,10 +154,18 @@ describe("gene entry", async () => {
     await utils.clickOn("section-bulk-add");
     await utils.typeInto("input-bulk-add", testGenes.join(","));
     await page.keyboard.press("Enter");
-    const userGeneHist = await cxgActions.getAllHistograms(
-      "histogram-user-gene"
+
+    // these load asynchronously, so we need to wait for each histogram individually
+    const elements = await Promise.all(
+      testGenes.map(g => page.waitForSelector(`[data-testid='histogram-${g}']`))
+    );
+    const idHandles = await Promise.all(
+      elements.map(el => el.getProperty("id"))
+    );
+    const ids = await Promise.all(idHandles.map(h => h.jsonValue()));
+    expect(ids.map(id => id.substring("histogram_".length))).toEqual(
+      expect.arrayContaining(testGenes)
```

Comment:
Please move this code to `getAllHistograms` in `e2e/puppeteerUtils.js` and have e2e.test.js continue to use the helper function.